### DISCUSSION TITLE
【教程】关于绳网帖子排版的小教程

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#999] [其他]
![image](https://github.com/user-attachments/assets/38bd3686-9c1a-43a8-bc2e-2a8b29f5d649)
这里是[具体教程链接](https://markdown.com.cn "Markdown官方教程")，以下是~~简单~~教程（共十条）（选了一些个人认为实用的操作）
本教程不包含html的教程因为好像被ban掉了（抑或是我设备的问题？）

******

## 0. 输入文字
 - 例
```
文字
```
- 效果

> 文字

***

## 1. 在文字前输入不同数量的“#”可以改变字的大小（数量越多字越小）
 - 例
```
# 山
## 里
### 灵
#### 活
##### 的
###### 狗
``` 
- 效果

> # 山
> ## 里
> ### 灵
> #### 活
> ##### 的
> ###### 狗

******

## 2. 在文字两侧分别输入一个“*”以表示斜体
 - 例
```
*文字*
```
- 效果

> *文字*

***

## 3. 在文字两侧分别输入两个“*”以表示加粗
 - 例
```
**文字**
``` 
- 效果

> **文字**

******

## 4. 依此类推
 - 例
```
***文字***
``` 
- 效果

> ***文字***

******

## 5. 在文字之前和之后的行上使用三个”`“以方便演示和复制
 - 例
~~~
```
文字
```
~~~
- 效果

> ```
> 文字
> ```

******

## 6. 输入三个或三个以上的”*“并上下各空一行以表示分隔符
 - 例
```
文字

***

文字
``` 
- 效果

> 文字
>
> ***
> 
> 文字

******

## 7. 用 ` [链接名](地址 "鼠标悬浮时显示的文字") ` 来表示一个链接（注意切换中英符号）
 - 例
```
[这是一个链接](https://www.github.com "一个链接")
``` 
- 效果

> [这是一个链接](https://www.github.com "一个链接")

******

## 8. 将传上来的图片形成的一串字符用英文中括号括起并后跟一个括有链接的括号以制作一个图片链接
 - 例
```
[![bb3ed05a740a80f71181a52058966793](https://github.com/user-attachments/assets/f78f518d-921b-4470-b252-650479eb7caa)](https://www.bilibili.com/video/BV1fA4m1P7qK/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "大香蕉一条大香蕉")

``` 
- 效果

> [![bb3ed05a740a80f71181a52058966793](https://github.com/user-attachments/assets/f78f518d-921b-4470-b252-650479eb7caa)](https://www.bilibili.com/video/BV1fA4m1P7qK/?share_source=copy_web&vd_source=722b2d6c4a0efb2b0fc1aa3c34f95a9d "大香蕉一条大香蕉")

******

## 9. 在文字两端输入两个“~”以实现”划掉“的效果
 - 例
```
我就是神~~金~~
``` 
- 效果

> 我就是神~~金~~

******

## 10. 使用“|”以创建表格，用三个及三个以上的“-”以表示列标题
 - 例
```
|米哈游|库洛|鹰角|完美世界|
|---|---|---|---|
|原神|战双|舟|幻塔|
|崩铁|鸣潮|明日方舟终末地|异环|
``` 
- 效果

> |米哈游|库洛|鹰角|完美世界|
> |---|---|---|---|
> |原神|战双帕弥什|明日方舟|幻塔|
> |崩坏：星穹铁道|鸣潮|明日方舟：终末地|异环|

******

# 以上是我所认为的较为实用的技巧，感谢所有看到这里的绳匠们

**Comment by Drs1145** [comment]
非常好的内容，使我的键盘旋转。唯一的缺点是没有帖子收藏功能

**Reply by xingruxuewji** [reply]
你可以直接搜索MarkDown教程
[MarkDown教程](https://www.runoob.com/markdown/md-tutorial.html "这是MarkDown教程")

**Comment by capacitor1** [comment]
🤔其实markdown还有一些高级功能，但是GitHub都不支持。。

比如TOC目录，html的关键标签比如script，video

**Comment by user1-cloud** [comment]
> 非常好的内容，使我的键盘旋转。唯一的缺点是没有帖子收藏功能

@Drs1145 
# 可以直接收藏Github页面(‾◡◝)

**Comment by xingruxuewji** [comment]
> 非常好的内容，使我的键盘旋转。唯一的缺点是没有帖子收藏功能

@Drs1145

你可以直接搜索MarkDown教程
[MarkDown教程](https://www.runoob.com/markdown/md-tutorial.html)

**Comment by FeiXiao-zm** [comment]
**原来如此！**

**Comment by 10kaihua** [comment]
太棒了，我正需要这个，非常有用的技巧

**Comment by LHk416** [comment]
太棒了可惜没有收藏

**Comment by themagicmoon** [comment]
**我还在想应用为什么点粗字体出来是四个星- 号，感谢大佬**

**Comment by Kamisato100** [comment]
***非常好教程，使我邦布旋转***

**Comment by FIREFLY26710-WEB** [comment]
# 哇都是干货，收藏了以备不时之需

**Comment by Inter-Knot-User** [comment]
**这个帖子我收藏了**

**Comment by su2569** [comment]
# 6啊

**Comment by szlx2333** [comment]
_收藏以备不时之需_

**Comment by sfhjhsdfg** [comment]
> [!TIP]
>翻了翻github[这也许有用](https://docs.github.com/zh/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

**Comment by FengLizhir** [comment]
谢谢大佬，使我能在绳网快乐的冲浪

**Comment by Bochiii984** [comment]
# 我
## 喜
### 欢
#### 你

**Comment by GUANGYINGZHIXIA** [comment]
很好的教程，使我的邦布旋转
